src/main/java/employee/data/Employee.java

```diff
@@ -72,6 +74,20 @@ public boolean isAddedDateEmpty() {
         return addedDate == null;
     }
 
+    @JsonIgnore
+    public boolean isISODate(){
+        try{
+            DateTimeFormatter formatter = DateTimeFormatter.ISO_LOCAL_DATE_TIME; //ofPattern("yyyy-MM-dd'T'HH:mm");
+
+            LocalDateTime.parse(addedDate, formatter);
+
+            return true;
+        }
+        catch (Exception e){
```

Comment:
is the `try/catch` here also bad practice @george-haddad

Reply:
In essence yes,